### PR TITLE
Update integration setups to use libcompose events

### PR DESCRIPTION
**Pull request by vdemeester**
Now that docker/libcompose#55 is merged, use it 🐒.

- No more sleeps !
- Update the `integration/fixture/file/simple.toml` to make sure it's gonna work (see #56)

🐸

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

**Comment by EmileVauge** [comment]
@vdemeester, cool :)
Don't we have to update the `Godeps.json` file?
Can we remove other sleep like [here](https://github.com/EmileVauge/traefik/blob/master/integration/marathon_test.go#L17) ?

File: integration/integration_test.go [hunk 1]
```diff
@@ -103,33 +98,35 @@ func (s *BaseSuite) createComposeProject(c *check.C, name string) {
 	c.Assert(err, checker.IsNil)
 	s.composeProject = composeProject
 
-	s.listenChan = make(chan project.ProjectEvent)
+	s.started = make(chan bool)
```
Comment (EmileVauge) [inline integration/integration_test.go]: We should find a defer close(s.started) somewhere (and also for other channels) :)

Comment (vdemeester) [inline integration/integration_test.go]: Hum je le fais ailleurs, mais soit, je vais te mettre des `defer` :stuck_out_tongue_closed_eyes:

**Comment by vdemeester** [comment]
>  Don't we have to update the `Godeps.json` file?

Yes, and I sure does :wink: (it's just github that don't like to show the actual diff but it's there).

File: integration/integration_test.go (hunk 1 above)
Comment (EmileVauge) [inline integration/integration_test.go]: Ah OK, if that's done elsewhere I'm OK with that :)

**Comment by EmileVauge** [comment]
@vdemeester ops, github tricked me ^^

**Comment by EmileVauge** [comment]
LGTM!

**Pull request merged**